.changeset/curly-crews-wait.md

```diff
@@ -0,0 +1,38 @@
+---
+'sku': major
+---
+
+Remove automatic configuration of Storybook
+
+**BREAKING CHANGE**
+
+Sku no longer configures your `.storybook/main.ts` file for you. If you are using Storybook, you will need to configure it yourself. See [`sku`'s Storybook documentation][storybook docs] for more information.
+
+**MIGRATION GUIDE**:
+
+Minimal Storybook configuration:
+
+```ts
+// .storybook/main.ts
+import { babel, webpackFinal } from 'sku/config/storybook';
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+export default {
+  stories: ['../src/**/*.stories.tsx'],
+  framework: '@storybook/react-webpack5',
+  core: {
+    builder: {
+      name: '@storybook/builder-webpack5',
```

Comment:
Just small fyi it looks like when running `storybook upgrade` it performs a framework migration which moves the builder under the `framework` object... It seemed to work with either one... Is the migration a bad migration or should we be using that?

<img width="735" alt="Screenshot 2024-06-26 at 10 24 25 AM" src="https://github.com/seek-oss/sku/assets/53756558/1e0ea845-1ea3-4051-9548-dc3b95cec765">

Reply:
This contradicts [the config structure in their docs](https://storybook.js.org/docs/builders/webpack#configure) 😡. I don't know who to believe.

Not sure why it's detecting a new framework migration though. Maybe it would be better if we just advised installing storybook 7 deps when doing the migration, then moving to storybook 8 separately, potentially via the CLI. Thoughts?

Reply:
I do prefer this syntax though:

```
  framework: {
    name: '@storybook/react-webpack5',
    options: {
      builder: {
        fsCache: true,
      },
    },
  },
```

- No reference to `@storybook/builder-webpack5`
- No `core` property, so less nesting

Despite the differences between it and the docs, I'd rather go with the simpler config.